Makefile

```diff
@@ -185,4 +186,9 @@ release:
 	@echo ""
 	@$(MAKE) _release
 
-
+.PHONY: docs
+docs:
+	@echo "=> Generate YAML documenation in ${DOCS_OUT}"
```

Comment:
Typo here in "documenation", should be "documentation"

Reply:
:man_facepalming: